src/Atmos/Model/radiation.jl

```diff
@@ -54,12 +55,12 @@ function flux_radiation!(m::StevensRadiation, flux::Grad, state::Vars,
   z = aux.orientation.Φ/grav
```

Comment:
```suggestion
  z = altitude(m.orientation, aux)
```
(Let's try to use the altitude function consistently, to isolate where this is computed.)